validate/validate.go

```diff
@@ -903,14 +903,14 @@ func namespaceValid(ns rspec.LinuxNamespace) bool {
 	case rspec.UserNamespace:
 	case rspec.CgroupNamespace:
 	default:
-		return false
+		return fmt.Errorf("invalid namespace type %s", ns.Type)
 	}
 
 	if ns.Path != "" && !filepath.IsAbs(ns.Path) {
-		return false
+		return fmt.Errorf("%v is not an absolute path", ns.Path)
```

Comment:
maybe fmt.Errorf(path %v of namespace %v is not absolute, ns.Type, ns.Path) better?

Reply:
Fine.